Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.10-alpine as builder
+FROM hexpm/elixir:1.11.0-erlang-21.3.8.21-alpine-3.13.1 as builder
```

Comment:
Is there a .tool-versions file? If so, it should also be updated with these versions.

Reply:
There is, and awkwardly, it specifies a newer Erlang version than what's provided by this base image. I'm going to try a different image&mdash;there are new builds as of this morning&mdash;and see about using the most recent versions of everything.

Reply:
See https://github.com/mbta/api_checker/pull/23/commits/bdcb880013e4254a20132aa179fc9023139fad7f. I bumped Erlang to 23.x, hopefully that won't be an issue but let me know if I should keep it on 22.

Reply:
Test failures imply that maybe Erlang 23 is too ambitious. https://semaphoreci.com/mbta/api_checker/branches/idw-image-deploy/builds/3 I'll see about backing out to 22 for now.

Reply:
Weird, that failure doesn't seem to make any sense for Erlang 23 🤷

Reply:
Got another failure on https://github.com/mbta/api_checker/pull/23/commits/e6c6c2e006b4a6f5a83983b6ba46d9f908dd269b. I'm officially out of my depth here. Any recommendations?

Reply:
Looks like it's on `main` too: https://semaphoreci.com/mbta/api_checker/branches/main/builds/2

This is likely caused by the dates we use for holiday checking falling out of the API: I'll update the code with some new dates.